Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageEx.Members.cs

```diff
@@ -51,6 +51,11 @@ public partial class ImageEx
         public static readonly DependencyProperty DecodePixelWidthProperty = DependencyProperty.Register(nameof(DecodePixelWidth), typeof(int), typeof(ImageEx), new PropertyMetadata(0));
 
         /// <summary>
+        /// Identifies the <see cref="IsCacheEnabled"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty IsCacheEnabledProperty = DependencyProperty.Register(nameof(IsCacheEnabled), typeof(bool), typeof(ImageEx), new PropertyMetadata(true));
```

Comment:
Whilst I have nothing against the default being changed to IsCacheEnabled to true, this is a significant change as previous it would always have a default value of false.